services/api/integration-tests/proxy.go

```diff
@@ -42,4 +46,37 @@ func (s *proxyTestSuite) TestProxy() {
 
 		require.NoError(t, err)
 	})
+
+	s.T().Run("should register chain connected to a protected rpc npc and create proxy to the node", func(t *testing.T) {
```

Comment:
This is copied from existing commit to private repo. https://github.com/Consensys/orchestrate-priv/commit/4ebf9a4e#diff-cbbc585e82a59d16bf5cd7be95fa6862ff405acec4899d28b97aa74288819b09R50